android/app/src/main/java/com/mattermost/rnbeta/MainActivity.java

```diff
@@ -29,6 +35,19 @@ public void onConfigurationChanged(Configuration newConfig) {
         }
     }
 
+    @Override
+    // This can be removed once https://github.com/facebook/react-native/issues/32628 is solved
+    public void onWindowFocusChanged(boolean hasFocus) {
+        super.onWindowFocusChanged(hasFocus);
+        MattermostManagedModule instance = MattermostManagedModule.getInstance();
+        if (instance != null) {
+            WritableMap data = Arguments.createMap();
+            data.putString("appState", hasFocus ? "active" : "background");
+            instance.sendEvent("windowFocusChanged", data);
+            Log.d("ReactNative", String.format("Window has focus %s", hasFocus));
```

Comment:
1. There is ko equivalent for iOS not is needed.
2. Is not a replacement, the AppState module is suppose to emit this event as `blur` and `focus` instead of `change` as per the rn docs which was introduced with RN 66 i believe. This events are not being emitted thus the reference to the guthub issue.

I had two alternatives,submit a PR to RN, build our own custom Android RN build and use it as RN for android uses a pre-built aar or add the event emitter here

There is a third option that will allow us to build RN for android and patch the module, but that increases android build time x3 as well as forces developers to install the ndk packages, so I went with the simplest option